src/components/dialogs/create-contingency-list-dialog.js

```diff
@@ -248,6 +311,21 @@ export const CreateContingencyListDialog = ({ open, onClose }) => {
                             label={<FormattedMessage id="FORM" />}
                         />
                     </RadioGroup>
+                    {contingencyListType === ContingencyListType.SCRIPT ? (
```

Comment:
Maybe use a switch here ? We gonna have 3 options in the next PR

Reply:
In the next PR is used this type of logic :
```
{type === AAA && <componentAAA>}
{type === BBB && <componentBBB>}
{type === CCC && <componentCCC>}
```
This change will be made in the next PR. For the moment, and with only two different components, I use the same thing as in filters.

Reply:
Ok, it's fine